wp_api/src/login/url_discovery.rs

```diff
@@ -581,13 +614,14 @@ pub enum ParseApiRootUrlError {
 #[derive(Debug, thiserror::Error, uniffi::Error)]
 pub enum FetchApiDetailsError {
     #[error(
-        "Request execution failed!\nStatus Code: '{:?}'.\nResponse: '{}'",
+        "Request execution faiasdfled!\nStatus Code: '{:?}'.\nResponse: '{}'",
```

Comment:
```suggestion
        "Request execution failed!\nStatus Code: '{:?}'.\nResponse: '{}'",
```

This is probably something you were testing and forgot to change it back? Or random typo 🙈